tests/TESTFPC1.MLC

```diff
@@ -82,7 +85,8 @@ LHT1     DC    LH'-567678867789987898'
          LA    R2,&VAL
          LA    R3,L'&VAL
          CALL  CVTTOHEX,((R2),DHEX,(R3))
-         WTO   MF=(E,WTOMSG)
+*        WTO   MF=(E,WTOMSG)                             #540
+         PUT   SYSOUT,WTOMSG                             #540
```

Comment:
Another WTO in immediately following IF .. ENDIF. Also, since the WTO is immediately followed by ABEND, should close the DCB.